test/c3/appframework/appframework_test.go

```diff
@@ -553,4 +554,158 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV2, false, "enabled", false, true)
 		})
 	})
+
+	Context("Single Site Indexer Cluster with SHC (C3) with App Framework", func() {
+		It("appfwint, c3, appframework: can deploy a C3 SVA with App Framework enabled for manual update", func() {
```

Comment:
Should `appfwint` be replaced with `integration`? cc: @pdhanoyasplunk

Reply:
Thanks for catching it. Yes we should replace it with `integration`.